src/annotator/anchoring/html.ts

```diff
@@ -1,18 +1,20 @@
+import type {
+  RangeSelector,
+  Selector,
+  TextPositionSelector,
+  TextQuoteSelector,
+} from '../../types/api';
+
 import { RangeAnchor, TextPositionAnchor, TextQuoteAnchor } from './types';
 
-/**
- * @typedef {import('../../types/api').RangeSelector} RangeSelector
- * @typedef {import('../../types/api').Selector} Selector
- * @typedef {import('../../types/api').TextPositionSelector} TextPositionSelector
- * @typedef {import('../../types/api').TextQuoteSelector} TextQuoteSelector
- */
+type Options = {
+  hint?: number;
+};
 
-/**
- * @param {RangeAnchor|TextPositionAnchor|TextQuoteAnchor} anchor
- * @param {object} [options]
- *  @param {number} [options.hint]
- */
-async function querySelector(anchor, options = {}) {
+async function querySelector(
+  anchor: RangeAnchor | TextPositionAnchor | TextQuoteAnchor,
+  options: Options
```

Comment:
Codecov was complaining about this line not being covered while having a default value on `options`.

I have realized it's a not-exposed function, and here it's always called with an options object, so it was safe to remove the default value.

Reply:
This is fine, although using a lint suppression to tell code coverage to ignore "simple" default arguments (eg. empty objects) is also a perfectly acceptable thing to do.